src/WebJobs.Script.WebHost/Diagnostics/SystemTraceWriter.cs

```diff
@@ -53,6 +54,11 @@ public override void Trace(TraceEvent traceEvent)
                     functionName = value.ToString();
                 }
 
+                if (traceEvent.Properties.TryGetValue(ScriptConstants.TracePropertyHostVersionKey, out value) && value != null)
```

Comment:
See my comments below... I really think we can simplify this and lower the impact of adding this.